packages/core/src/App/Containers/Layout/header/__tests__/default-mobile-links.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import DefaultMobileLinks from '../default-mobile-links';
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Popover: () => <div>mockedPopover</div>,
+    };
+});
+jest.mock('App/Components/Routes', () => jest.fn(() => 'mockedBinaryLink'));
+jest.mock('../show-notifications', () => jest.fn(() => 'mockedShowNotifications'));
+jest.mock('../traders-hub-onboarding', () => jest.fn(() => 'mockedTradersHubOnboarding'));
+
+describe('DefaultMobileLinks', () => {
+    const mock_props: React.ComponentProps<typeof DefaultMobileLinks> = {
+        handleClickCashier: jest.fn(),
+    };
+
+    it('should render the component', () => {
+        render(<DefaultMobileLinks {...mock_props} />);
+        expect(screen.getByText('mockedTradersHubOnboarding')).toBeInTheDocument();
+    });
+
+    it('should display the cashier button', () => {
+        render(<DefaultMobileLinks {...mock_props} />);
+        expect(screen.getByText('Cashier')).toBeInTheDocument();
```

Comment:
getByRole has a higher priority 
![image](https://github.com/binary-com/deriv-app/assets/93518187/b2237390-4a51-410b-9b65-b85ca371b906)

```suggestion
        expect(screen.getByRole('button', { name: 'Cashier' })).toBeInTheDocument();
```